examples/qml_minimal/CMakeLists.txt

```diff
@@ -36,6 +42,10 @@ endif()
 get_target_property(QMAKE Qt::qmake IMPORTED_LOCATION)
 # ANCHOR_END: book_cmake_find_qmake
 
+if(BUILD_WASM)
+    set(Rust_CARGO_TARGET wasm32-unknown-emscripten)
```

Comment:
guess this could be combined with the other BUILD_WASM block further up ?

eg have

```cmake
if(BUILD_WASM)
    # When building under emscripten tell CMake how to link using whole archive
    set(CMAKE_LINK_LIBRARY_USING_WHOLE_ARCHIVE "-Wl,--whole-archive" "<LINK_ITEM>" "-Wl,--no-whole-archive")
    set(CMAKE_LINK_LIBRARY_USING_WHOLE_ARCHIVE_SUPPORTED True)
    # Ensure Rust build for the correct target
    set(Rust_CARGO_TARGET wasm32-unknown-emscripten)
endif()
```

there is also the question over whether this should be global or in examples, for now lets also put it in the examples. So you'll need this same snippet in qml_features and demo_threading